pkg/api/types.go

```diff
@@ -123,11 +123,21 @@ type InputConfiguration struct {
 	// build-time dependencies for the project.
 	TestBaseImage *ImageStreamTagReference `json:"test_base_image,omitempty"`
 
+	CustomTestBaseImage *GitSourceReference `json:"custom_test_base_image,omitempty"`
+
 	// ReleaseTagConfiguration determines how the
 	// full release is assembled.
 	ReleaseTagConfiguration *ReleaseTagConfiguration `json:"tag_specification,omitempty"`
 }
 
+// GitSourceReference
+type GitSourceReference struct {
+	ContextDir     string `json:"contextDir,omitempty"`
+	Uri            string `json:"uri,omitempty"`
```

Comment:
The URI and Ref are provided from Prow in the job spec and do not need to be exposed here